### DISCUSSION TITLE
Workflow schema

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#59] [General]
Hi @gregdenay ,

not sure you noticed, but I have added a schematic overview of FooDMe2 to the README.md - https://github.com/bio-raum/FooDMe2/tree/dev

I am using a similar design for GABI - https://github.com/bio-raum/gabi

Although these needs to be standardized a bit more (font size etc). Are you fine with this, or should we do this some other way?

**Comment by gregdenay** [comment]
It looks good and it is a good idea. 
The [bio-raum logo](https://www.flaticon.com/free-icon/biological_3839520) is just a placeholder icon I had found on the quick, an attribution would be nescessary. We could also use the Foodme logo instead.

**Reply by marchoeppner** [reply]
I am not sure the psychedelic FooDMe2 Logo needs to be spread more widely ;)

I am open to suggestions; though the current bio-raum logo does look pretty nice in its clean and unexcited design.

**Reply by gregdenay** [reply]
I feel offended.
I added a profile sheet to the bio-raum organization, with the credits for the logo. We can copy the credit line wherever we use it and it should be enough. Is that ok with you?
Feel free to edit the bio-raum profile if you want to !
